README.md

```diff
@@ -1,9 +1,13 @@
 # `node-gyp` - Node.js native addon build tool
 
-`node-gyp` is a cross-platform command-line tool written in Node.js for compiling
-native addon modules for Node.js. It bundles the [gyp](https://gyp.gsrc.io)
-project used by the Chromium team and takes away the pain of dealing with the
-various differences in build platforms.
+`node-gyp` is a cross-platform command-line tool written in Node.js for
+compiling native addon modules for Node.js. It contains a fork of the
+[gyp](https://gyp.gsrc.io) project that was previously used by the Chromium
+team and takes away the pain of dealing with the various differences in build
+platforms.
+
+Note that `node-gyp` is _not_ used to build Node.js itself, for information
+on that, see https://github.com/nodejs/node.
```

Comment:
It feels that we should be pointing to more specific information if we say "for information on that" but I'm not aware that we have much (any?) documents in https://github.com/nodejs/node that talk about `gyp`. Maybe [BUILDING.md](https://github.com/nodejs/node/blob/master/BUILDING.md) is the most suitable link?

Reply:
I was weasely wording on purpose, because I don't think the node-gyp project should encode nodejs/node documentation locations into its README. I think we should say _something_ about it, because "node gyp" makes it sound like it's node's gyp.... and it isn't. How about I just terminate the sentence at the `,`, removing the entire ", for information ..."?

Reply:
> How about I just terminate the sentence at the `,`, removing the entire ", for information ..."?

👍 Works for me.